parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -340,6 +344,18 @@ ensureTeleportd() {
     systemctlEnableAndStart teleportd || exit $ERR_SYSTEMCTL_START_FAIL
 }
 
+ensureArtifactStreaming() {
+  systemctl start acr-mirror.service
```

Comment:
q for understanding: are we enabling this service during VHD build?

Reply:
it is not enabled during VHD build. we only want to enable it when artifact streaming is enabled

Reply:
right, so should we do a `systemctl enable acr-mirror.service` before starting it?

Reply:
right, pushed it :). thanks